### PR TITLE
docs: add foundations topic cards and iconography page

### PR DESCRIPTION
**Pull request by jannick-ux**
## Vitepress documentation update

The page of iconography is filled now. Overview cards for "Basics" are created. 

## Checklist

- [x] The added / edited code has been documented with [JSDoc](https://jsdoc.app/about-getting-started)
- [x] All changes are documented in the documentation app (folder `apps/docs`)
- [x] If a new component is added, at least one [Playwright screenshot test](https://github.com/SchwarzIT/onyx/actions/workflows/playwright-screenshots.yml) is added
- [x] A changeset is added with `npx changeset add` if your changes should be released as npm package (because they affect the library usage)

### CODE REVIEW

**Review by BoppLi — CHANGES_REQUESTED** [review]

File: apps/docs/src/basics/iconography.md [hunk 1]
```diff
@@ -1,3 +1,68 @@
 # Iconography
 
+:::warning Attention
```
Comment (larsrickert) [inline apps/docs/src/basics/iconography.md:3]:
```suggestion
::: warning Attention
```

File: apps/docs/src/basics/iconography.md [hunk 2]
```diff
@@ -1,3 +1,68 @@
...
+Check out the whole icon palette in the [resources](/resources/icons) documentation.
+Feel free to browse through all the assets there.
+:::
+
+Our system icons are designed to be simple with a touch of detail. They are characteristic yet easy to understand.
```
Comment (larsrickert) [inline apps/docs/src/basics/iconography.md:8]:
```suggestion
Our system icons are designed to be simple with a touch of detail. They are characteristic, yet easy to understand.
```

File: apps/docs/src/basics/iconography.md [hunk 3]
```diff
@@ -1,3 +1,68 @@
...
+
+![Image](/assets/icons-overview.png)
```
Comment (larsrickert) [inline apps/docs/src/basics/iconography.md:10]: The "Image" alt text is used for screen readers so its best to set it to something meaningful for each image that describes its content

File: apps/docs/src/basics/iconography.md [hunk 4]
```diff
@@ -1,3 +1,68 @@
...
+
+## Grid and keyline shapes
+
+The square format with the 1px grid is the foundation for all of our system icons. This ensures the look of the set in terms of thickness, proportion, shape and weight are the same. The grid helps to design a unified set of icons and still be as flexible as possible.
+
+![Image](/assets/icons-grid.png)
+
+## Basic layout
+
+the icons are created on a base grid of 32x32px and then reduced or enlarged to the appropriate size. The grid comes with a padding of 2 px. The padding should only be exceeded if it is necessary to give additional visual weighting or if a certain element is required to give the icon its characteristic or meaning.
```
Comment (larsrickert) [inline apps/docs/src/basics/iconography.md:20]:
```suggestion
The icons are created on a base grid of 32x32px and then reduced or enlarged to the appropriate size. The grid comes with a padding of 2px. The padding should only be exceeded if it is necessary to give additional visual weighting or if a certain element is required to give the icon its characteristic or meaning.
```

File: apps/docs/src/basics/iconography.md [hunk 5]
```diff
@@ -1,3 +1,68 @@
...
+
+![Image](/assets/icons-layout-1.png)
+
+![Image](/assets/icons-layout-2.png)
+
+The key lines shapes ensure a certain consistency in shape and proportion in the overall icon set. In exceptional cases and if the design of the icon does not allow otherwise, the design can go beyond the shape.
```
Comment (larsrickert) [inline apps/docs/src/basics/iconography.md:26]: The wording "The key lines shapes" is a little off or don't I understand the sentence? :D

File: apps/docs/src/basics/iconography.md [hunk 6]
```diff
@@ -1,3 +1,68 @@
...
+
+![Image](/assets/icons-layout-3.png)
+
+## Stroke
+
+The weighting of different icons should be the same as possible and should not look heavier or lighter. To make this possible, only a 2px stroke is used.
+
+![Image](/assets/icons-stroke-1.png)
+
+The distance between parallel strokes is at least 2px. There are a few exceptions which occurs when the icon is complex or has a certain density of line
```
Comment (larsrickert) [inline apps/docs/src/basics/iconography.md:36]:
```suggestion
The distance between parallel strokes is at least 2px. There are a few exceptions which occurs when the icon is complex or has a certain density of line.
```

File: apps/docs/src/basics/iconography.md [hunk 7]
```diff
@@ -1,3 +1,68 @@
...
+
+## Angles
+
+The basic angle is 45 °. If necessary, the increments of 15° and 5° are the next steps.
+
+![Image](/assets/icons-angles.png)
+
+## Corners
+
+Unified corner radius ensure the element is recognizable and in harmony with the set. 2 px is the standard radius and can be increased in further steps of two.
```
Comment (larsrickert) [inline apps/docs/src/basics/iconography.md:48]:
```suggestion
Unified corner radius ensure the element is recognizable and in harmony with the set. 2px is the standard radius and can be increased in further steps of two.
```

File: apps/docs/src/basics/iconography.md [hunk 8]
```diff
@@ -1,3 +1,68 @@
...
+
+![Image](/assets/icons-corners.png)
+
+## Sizes
+
+All icons are created as vector graphics and due to the base unit, the icons are limited to specific sizes.
+
+In most cases the default size is 24px Square, but we also use a smaller and a larger variant based on the system patterns.
```
Comment (larsrickert) [inline apps/docs/src/basics/iconography.md:56]:
```suggestion
In most cases the default size is 24x24px, but we also use a smaller and a larger variant based on the system patterns.
```

**Review by larsrickert — CHANGES_REQUESTED** [review]

**Review by larsrickert — COMMENTED** [review]
Is it just me or does the preview images look a little low resolution?
image

File: apps/docs/src/basics/iconography.md (hunk 8 above)
Comment (JoCa96) [reply apps/docs/src/basics/iconography.md:56]:
```suggestion
The default size is 24x24px, but we also use a smaller and a larger variant based on the system patterns.
```

File: apps/docs/src/basics/iconography.md (hunk 5 above)
Comment (jannick-ux) [reply apps/docs/src/basics/iconography.md:26]: This is kept from the already existing guidelines, that were definied earlier (not by myself). It is about the shapes, that are visualizing the the key lines. And those key-line-shapes ensure certain consistency...

**Comment by jannick-ux** [comment]
@larsrickert about the low resolution: I thought the exact same thing. While exporting the pngs in Figma, I've chosen original size export. Maybe it is better to use "x2" size export. This would apply to all images that are used.

**Review by BoppLi — APPROVED** [review]

**Pull request merged**